### PR TITLE
Optional category field

### PR DESCRIPTION
**Pull request by allysonyoung**
adds an optional category field (denoted by dashes) for range quizzes

### CODE REVIEW

**Comment by nelsonpecora** [comment]
...tests?

File: parser.pegjs [hunk 1]
```diff
@@ -109,9 +114,17 @@ Range
...
+    if (c) {
+      obj.category = c.category.trim()
+    }
+
     return obj;
   }
 
+// category
+Category
+  = ws? dash c:Words? dash { return { category: c }; }
```
Comment (nelsonpecora) [inline parser.pegjs:126]: does this allow commas in category names?

Comment (allysonyoung) [reply parser.pegjs:126]: yes, just checked

Comment (nelsonpecora) [reply parser.pegjs:126]: coooool!

**Comment by nelsonpecora** [comment]
👍  👍  👍  looks good, thanks! feel free to merge and release a new minor version

**Pull request merged**